### ISSUE TITLE
Make library cross-platform

### ISSUE THREAD

**Issue by aadibajpai** [mrtaalebi/sponus#3]
You might want to use SwSpotify (https://github.com/SwagLyrics/SwSpotify) to make sure it runs on all OSes.

**Comment by mrtaalebi** [comment]
Yeah. I saw that repository before, Thank you it's in the schedule!
